lib/css/CssParser.js

```diff
@@ -597,6 +620,37 @@ class CssParser extends Parser {
 				}
 				return end;
 			},
+			function: (input, start, end) => {
```

Comment:
Can we get JSDoc annotations here. Looks easy enough to have typesafety here.

Reply:
We already have it, that is why no typescript problems, it is callback, and callbacks are defined in `parser`, so typescript knows `input` is `string` and `start` and `end` are number